images/dind/node/openshift-sdn-node.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source /usr/local/bin/openshift-dind-lib.sh
+# Should set OPENSHIFT_NETWORK_PLUGIN
+source /data/dind-env
+
+function have-token() {
+  local master_dir=$1
+
+  [[ -s "${master_dir}/openshift-sdn.token" ]]
+}
+
+function openshift-sdn-node() {
+  local config_dir=$1
+  local master_dir=$2
+  local kube_config="${config_dir}/node.kubeconfig"
+  local sdn_kube_config="${config_dir}/sdn-node.kubeconfig"
+
+  os::util::wait-for-condition "kubernetes token" "have-token ${master_dir}" "120"
+  token=$(cat ${master_dir}/openshift-sdn.token)
+
+  # Take over network functions on the node
+  rm -Rf /etc/cni/net.d/80-openshift-network.conf
+
+  # use either the bootstrapped node kubeconfig or the static configuration
+  if [[ ! -f "${kube_config}" ]]; then
+    # use the static node config if it exists
+    # TODO: remove when static node configuration is no longer supported
+    for f in ${config_dir}/system*.kubeconfig; do
```

Comment:
I'd add a `shopt -s failglob` command here, or an additional check after the `for` loop to make sure `$kube_config` exists.  (I was surprised to find that `oc config --config=non-existent-file view --flatten` does not fail.)

Reply:
@Miciah yeah, I was surprised too.  Anyway, added a post-for check.